config/config.go

```diff
@@ -530,6 +531,7 @@ func NewConfig() (c *Config) {
 		KialiFeatureFlags: KialiFeatureFlags{
 			IstioInjectionAction: true,
 			IstioUpgradeAction:   false,
+			SkipSubsetValidation: false,
```

Comment:
this should be moved under the "Validations" struct since it is a feature flag specifically involving validations.